tests/Oro/Tests/ORM/AST/Query/Functions/FunctionsTest.php

```diff
@@ -24,6 +24,9 @@ public function testDqlFunction(array $functions, $dql, $sql, $expectedResult)
         $configuration = $this->entityManager->getConfiguration();
 
         foreach ($functions as $function) {
+            if (isset($function['minimumVersion'])) {
+                $this->skipIfNotMinimumVersion($function['minimumVersion']);
```

Comment:
doctrine-extensions must work same way on all supported DBs.
Please, implement for all supported platforms.